client/src/helpers/strings/wise.ts

```diff
@@ -71,25 +68,23 @@ const strings: PlatformStrings = {
     Optionally, provide a recipient address below to receive funds in another wallet. Submit transaction to start your order
   `,
   INSTRUCTION_DRAWER_STEP_THREE: `
-    Click 'Send' and complete the payment on any UPI ID linked to your Wise bank account. Ensure you have email notifications from Wise InstaAlerts enabled
+    Click 'Send' and complete the payment from your Wise account. Ensure you have access to a dekstop to prove the payment later
   `,
   INSTRUCTION_DRAWER_STEP_FOUR: `
-    Continue through to validate email proof of transaction. Submit transaction containing proof to receive the requested USDC
+    Continue through to generate and validate proof of payment. Submit transaction containing proof to receive the requested USDC
   `,
 
   // Payment Requirements
   PAYMENT_REQUIREMENT_STEP_ONE: `
-    Send payment from your UPI ID linked to your Wise bank account. Do not send payment from UPI Lite or other bank accounts
+    Send payment from your ZKP2P registered Wise acount
   `,
   PAYMENT_REQUIREMENT_STEP_TWO: `
-    Wise InstaAlert email notifications are enabled for your account
+    Chose the right receiving currency (EUR) while sending the payment
```

Comment:
- `send and receive currency` perhaps?

Reply:
We aren't constraining the send currency, as long as the receiver receives their desired currency and amount.

Reply:
We should make this clear somewhere

Reply:
Added in 9468ea2. Although can make it clear somewhere else to make users aware of this flexibility.